### PR TITLE
Add basic navigation bar and css styling

### PR DESCRIPTION
**Pull request by shulijones**

### CODE REVIEW

**Review by otobin — COMMENTED** [review]
Looks great! Not sure if it follows the style guide because I'm still going through the style guide but your code is super neat! Love the color theme and top nav bar. The picture of your dog isn't showing up for me but it looks like you formatted it correctly.

File: portfolio/src/main/webapp/elsewhere.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```
Comment (otobin) [inline portfolio/src/main/webapp/elsewhere.html:1]: Super neat style!

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,16 +13,22 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
```
Comment (otobin) [inline portfolio/src/main/webapp/script.js:18]: Super cool facts!

**Review by kyracq — APPROVED** [review]
Your portfolio looks great and your code is organized and easy to understand!

File: portfolio/src/main/webapp/elsewhere.html (hunk 1 above)
Comment (shulijones) [reply portfolio/src/main/webapp/elsewhere.html:1]: Thank you!! Just so you know, I played around a bit and I think I fixed the image issue by removing the slash in front of images/ella-1.JPG.

**Review by nsri-google — APPROVED** [review]
LGTM.

**Review by anicr7 — APPROVED** [review]

**Pull request merged**